src/pages/endorsements.astro

```diff
@@ -0,0 +1,45 @@
+---
+// © 2024 Vlad-Stefan Harbuz <vlad@vladh.net>
+// SPDX-License-Identifier: Apache-2.0
+
+import Blob from "../components/Blob.astro";
+import Button from "../components/Button.astro";
+import EndorsementList from "../components/EndorsementList.astro";
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout>
+  <div class="edge-blobs">
+    <Blob kind="solid-fill-03" top="42%" left="-13rem"></Blob>
+    <Blob kind="grad-dots-06" top="37%" right="-5rem"></Blob>
+  </div>
+  <div class="container">
+    <Blob kind="solid-fill-05" top="20%" right="-16rem"></Blob>
+    <Blob kind="solid-fill-04" top="90%" left="-20rem"></Blob>
+    <Blob kind="grad-dots-03" top="10%" left="-15rem"></Blob>
+
+    <main id="main-content">
+      <h1>Endorsements</h1>
+      <p>See what lovely folks from across the Open Source ecosystem had to say about the Open Source Pledge.</p>
```

Comment:
Tighten up this way? We have @ThePledge on X, are we ready to start referring to ourselves this way sometimes on the website?

```suggestion
      <p>See what lovely folks across the Open Source ecosystem have to say about the Pledge.</p>
```